Gruntfile.js

```diff
@@ -28,42 +22,31 @@ grunt.initConfig( {
 	concat: {
 		options: { process: process },
 		"src-js": {
-			src: [
-				"src/intro.js",
-				"src/core/initialize.js",
-				"src/core/utilities.js",
-				"src/core/stacktrace.js",
-				"src/core/config.js",
-				"src/core/logging.js",
-				"src/core/onerror.js",
-				"src/core.js",
-				"src/test.js",
-				"src/assert.js",
-				"src/equiv.js",
-				"src/dump.js",
-				"src/export.js",
-				"src/outro.js",
-				"runner/fixture.js",
-				"reporter/intro.js",
-				"reporter/urlparams.js",
-				"reporter/html.js",
-				"reporter/diff.js",
-				"reporter/outro.js"
-			],
+			src: "dist/qunit.js",
 			dest: "dist/qunit.js"
```

Comment:
it seems we only need this for the process part. I'm ok to keep it for now, but we should replace this for anything more appropriate in a future. If we merge it this way, let's remind to open an issue to address this.